Microsoft.Toolkit.Uwp.UI.Controls/BladeView/BladeView.cs

```diff
@@ -133,6 +151,12 @@ private void CycleBlades()
 
             BladeClosed?.Invoke(this, blade);
             ActiveBlades.Remove(blade);
+
+            var lastBlade = ActiveBlades.LastOrDefault();
+            if (lastBlade != null && lastBlade.TitleBarVisibility == Visibility.Visible)
```

Comment:
Shouldn't check the titlebar visibility. This can be set to collapsed by the user but still want the last blade to be expanded